src/components/map/sectors/MapSectorsList.vue

```diff
@@ -1,38 +1,140 @@
+        <template
+            v-for="(countryEntries, countryId) in dataStore.vatglassesActivePositions.value"
+            :key="countryId"
+        >
+            <map-vatglasses-position
+                v-for="(position, positionId) in countryEntries"
+                :key="countryId + '-' + positionId"
+                :position="position"
+            />
+        </template>
+
+
+        <map-overlay
+            class="vatglasses-overlay"
+            :model-value="vatglassesPopupIsShown"
+            :settings="{
+                position: getCoordinates,
+                offset: [15, -15],
+            }"
+            :z-index="20"
+        >
+            <common-popup-block
+                class="aircraft-hover"
+            >
+                <template #title>
+                    Positions
+                </template>
+
+                <ul>
+                    <li
+                        v-for="(sector, index) in sectorsAtClick"
+                        :key="index"
+                    >
+                        {{ sector.vatglassesPositionId }} ({{ dataStore.vatglassesActivePositions.value[sector.countryGroupId][sector.vatglassesPositionId].callsign }}) {{ formatNumber(sector.altrangeMin ?? sector.min) }} - {{ formatNumber(sector.altrangeMax ?? sector.max) }}
+                    </li>
+                </ul>
+
+            </common-popup-block>
+        </map-overlay>
+    </template>
 </template>
 
 <script setup lang="ts">
 import VectorSource from 'ol/source/Vector';
 import type { ShallowRef } from 'vue';
-import type { Map } from 'ol';
+import type { Map, MapBrowserEvent } from 'ol';
 import { Fill, Stroke, Style } from 'ol/style';
-import MapSector from '~/components/map/sectors/MapSector.vue';
+import MapVatglassesPosition from '~/components/map/sectors/MapVatglassesPosition.vue';
 import VectorImageLayer from 'ol/layer/VectorImage';
+import { useStore } from '~/store';
+import MapSector from '~/components/map/sectors/MapSector.vue';
+
+import { initVatglasses } from '~/utils/data/vatglasses';
+
+import type { Pixel } from 'ol/pixel';
+import { useMapStore } from '~/store/map';
+
 
 let vectorLayer: VectorImageLayer<any>;
 const vectorSource = shallowRef<VectorSource | null>(null);
 provide('vector-source', vectorSource);
 const map = inject<ShallowRef<Map | null>>('map')!;
 const dataStore = useDataStore();
+const store = useStore();
+const mapStore = useMapStore();
+
 
 const firs = computed(() => {
     const list = dataStore.vatspy.value!.data.firs;
     const firs = list.map(fir => ({
         fir,
         atc: dataStore.vatsim.data.firs.value.filter(x => x.firs.some(x => x.boundaryId === fir.feature.id && (fir.icao === x.icao || (fir.callsign && fir.callsign === x.callsign)))) ?? [],
     }));
-
     return firs.filter((x, xIndex) => !firs.some((y, yIndex) => y.fir.icao === x.fir.icao && x.fir.feature.id === y.fir.feature.id && yIndex < xIndex));
 });
 
+
+interface FeatureProperties {
```

Comment:
We need a proper global type for that

Reply:
It is the type of the return of  feature.getProperties(), feature is a normal openlayers feature. Do you have an idea how to solve that?

Reply:
I have added the type VatglassesSectorProperties, please check if that is what you requested.